reference/constraints.rst

```diff
@@ -50,6 +50,7 @@ Validation Constraints Reference
    constraints/Currency
    constraints/Luhn
    constraints/Iban
+   constraints/Bic
```

Comment:
I think you mean updating the code examples, right? And yes, I agree with you then.